ucp/_libs/ucx_api_dep.pxd

```diff
@@ -240,21 +241,13 @@ cdef extern from "sys/epoll.h":
 
 
 cdef struct ucp_request:
-    bint finished
-    PyObject *future
-    PyObject *event_loop
-    PyObject *log_msg
-    PyObject *inflight_msgs
-    size_t expected_receive
-    int64_t received
+    bint finished  # Used by downstream projects such as cuML
+    int uid
+    PyObject *info
```

Comment:
Should these match [these]( https://github.com/openucx/ucx/blob/a21bdf4d06d7b645ad37844fcac2ab047df9071e/src/ucp/core/ucp_request.h#L102-L104 )?

Reply:
No, but that is a potential name clash! 
Renamed to `ucp_request => ucx_py_request` and moved its definition into the `ucx_api.pyx` file.